http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -152,3 +152,29 @@ jq --null-input \
 /anaconda/envs/py38_default/bin/jupyter kernelspec list | awk 'NR>1 {print $2}' | while read line; do jq -s add $line"/kernel.json" wsenv.json > tmpkernel.json && mv tmpkernel.json $line"/kernel.json"; done
 /anaconda/envs/azureml_py38/bin/jupyter kernelspec list | awk 'NR>1 {print $2}' | while read line; do jq -s add $line"/kernel.json" wsenv.json > tmpkernel.json && mv tmpkernel.json $line"/kernel.json"; done
 /anaconda/envs/azureml_py38_PT_and_TF/bin/jupyter kernelspec list | awk 'NR>1 {print $2}' | while read line; do jq -s add $line"/kernel.json" wsenv.json > tmpkernel.json && mv tmpkernel.json $line"/kernel.json"; done
+
+log 'Formatting and mounting persistent disk...'
+
+WORK_DIRECTORY='/home/jupyter/persistent_disk'
+
+# Mount persistent disk
+# The PD should be the only `sd` disk that is not mounted yet
+AllsdDisks=($(/usr/bin/lsblk --nodeps --noheadings --output NAME --paths | grep -i "sd"))
+FreesdDisks=()
+for Disk in "${AllsdDisks[@]}"; do
+    Mounts="$(/usr/bin/lsblk --noheadings --output MOUNTPOINT "${Disk}" | grep -vE "^$")"
+    if [ "${Mounts}" == "" ]; then
+        FreesdDisks+=("${Disk}")
+    fi
+done
+
+DISK_DEVICE_ID= basename ${FreesdDisks[0]}
```

Comment:
missing backtick?
```
DISK_DEVICE_ID=`basename ${FreesdDisks[0]}`
```
Not sure..

Reply:
I do not think so? I will know as soon as i can get this part of the script to actually run and not fail on the pre-existing steps 😡